### ISSUE TITLE
Mixed Episode

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11588]
**Preferred term label:**
Mixed Episode

**Synonyms**

**Definition (free text, please give PubMed ID)**
A specifier for an affective episode in which there is the concurrent presence of core symptoms of the opposite mood polarity. In the context of bipolar disorder or major depressive disorder, this is characterized by the presence of at least three symptoms of the opposite mood polarity during a manic, hypomanic, or depressive episode.

PMID: [28184334](https://pubmed.ncbi.nlm.nih.gov/28184334/)

**Parent term (use hpo.jax.org/app)**
- Abnormally increased volition HP:5200263
- Abnormal emotional state HP:0100851
- Abnormal diminished volition HP:0000745

Not sure, if Abnormally increased volition and Abnormal diminished volition should be the parent, alternatively only Abnormal emotional state.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0002-1118-4784

**Comment by SmartMonkey-git** [comment]
If this is a no-go, then I would suggest a new clinical modifier called "with mixed features"
